### ISSUE TITLE
Please stop showing new repositories by default

### ISSUE THREAD

**Issue by ludeeus** [hacs/integration#2150]
### Discussed in https://github.com/hacs/integration/discussions/2149

<div type='discussions-op-text'>

<sup>Originally posted by **mikesalz** August 17, 2021</sup>
Any time I go to the HACS screen in HA to see my installed HACS integrations, I am bombarded with a massive list of new integrations.  Yes, I can click the ellipses and Dismiss all new repositories.  But I shouldn't have to.  If I want to find a new integration, I can click the Explore button.  The act of browsing and selecting integrations should be something that someone chooses to do, not something that is forced on them.  Please and thank you!</div>

**Comment by ludeeus** [comment]
Discussion deleted;
1) Did not follow template
2) Duplicate of https://github.com/hacs/integration/discussions/1946
3) Blocked by https://github.com/hacs/integration/discussions/1951
